jjb/dynamic/scale-ci_install_azure.yml

```diff
@@ -0,0 +1,311 @@
+        set -eux
+
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
+        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        export PBENCH_SSH_PUBLIC_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PBENCH_SSH_PRIVATE_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        chmod 600 ${PRIVATE_KEY}
+
+        # Create inventory File:
+        echo "[orchestration]" > inventory
+        echo "${ORCHESTRATION_HOST}" >> inventory
+
+        export ANSIBLE_FORCE_COLOR=true
+        ansible --version
+        time ansible-playbook -vv -i inventory OCP-4.X/install-on-azure.yml
+    concurrent: true
+    description: |
+      Installs and configures OCP 4.x cluster on Azure.
+      This job is managed by https://github.com/openshift-scale/scale-ci-pipeline
+    disabled: false
+    name: ATS-SCALE-CI-OCP-AZURE-DEPLOY
+    node: scale-ci
+    parameters:
+    - string:
+        default: "root"
+        description: The user for the Orchestration host.
+        name: ORCHESTRATION_USER
+    - string:
+        default: ""
+        description: Host with the kubeconfig and oc client.
+        name: ORCHESTRATION_HOST
+    - password:
+        default: ""
+        description: Token to access private repo containing ssh keys.
+        name: SSHKEY_TOKEN
+    - bool:
+        default: false
+        description: destroys the cluster created using scale-ci-deploy.
+        name: OPENSHIFT_CLEANUP
+    - bool:
+        default: true
+        description: Installs OCP cluster on AWS.
+        name: OPENSHIFT_INSTALL
+    - bool:
+        default: true
+        description: Creates workload and infra nodes post install.
+        name: OPENSHIFT_POST_INSTALL
+    - bool:
+        default: true
+        description: Configures the cluster to enable running workloads.
+        name: OPENSHIFT_POST_CONFIG
+    - bool:
+        default: false
+        description: Enables debugging i.e ssh to the cluster nodes.
+        name: OPENSHIFT_DEBUG_CONFIG
+    - bool:
+        default: ''
+        description: OC client url.
+        name: OPENSHIFT_CLIENT_LOCATION
+    - string:
+        default: ''
+        description: Image to use for the ocp install.
+        name: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+    - string:
+        default: "v1"
+        description: openshift apiversion
+        name: OPENSHIFT_INSTALL_APIVERSION
+    - string:
+        default: ""
+        description: Path to the public key to add to each openshift host.
+        name: OPENSHIFT_INSTALL_SSH_PUB_KEY_FILE
+    - password:
+        default: ""
+        description: pull secret
+        name: OPENSHIFT_INSTALL_PULL_SECRET
+    - password:
+        default: ""
+        description: Registry token to talk to quay registry
+        name: OPENSHIFT_INSTALL_QUAY_REGISTRY_TOKEN
+    - string:
+        default: ""
```

Comment:
I don't think there is any issue with including the default here `registry.svc.ci.openshift.org`